### PR TITLE
Add configuration for not deployed profile

### PR DESCRIPTION
**Pull request by Tobianas**
Added changes to disable CSRF for localhost only
to make fab54d78 work.

JIRA:LIGHTY-213

### CODE REVIEW

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationDeployed.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2018-2021 PANTHEON.tech s.r.o. All Rights Reserved.
```
Comment (ihrasko) [inline lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationDeployed.java:2]: 2023

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationDeployed.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v10.html
+ */
+
```
Comment (ihrasko) [inline lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationDeployed.java:8]: Remove this empty line.

**Review by ihrasko — CHANGES_REQUESTED** [review]

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationDeployed.java (hunk 1 above)
Comment (Tobianas) [reply lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationDeployed.java]: fixed

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationDeployed.java (hunk 2 above)
Comment (Tobianas) [reply lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationDeployed.java]: removed

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationNotDeployed.java [hunk 3]
```diff
@@ -42,5 +44,4 @@ protected SecurityFilterChain auth0FilterChain(HttpSecurity httpSecurity) throws
                 .disable()
```
[the remark this answers is not in the document]
Comment (ihrasko) [reply lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/SecurityConfigurationNotDeployed.java:44]: ignore

**Review by ihrasko — APPROVED** [review]

**Pull request merged**